app/src/main/java/com/cornellappdev/uplift/ui/screens/GymDetailScreen.kt

```diff
@@ -180,31 +183,74 @@ fun GymDetailScreen(
                 fontFamily = montserratFamily
             )
 
-            // CLOSED
-            if (gym != null && (gym!!.hours[day] == null || !isOpen(gym!!.hours[day]!!))) {
-                Box(
-                    modifier = Modifier
-                        .align(Alignment.BottomCenter)
-                        .graphicsLayer {
-                            translationY = -0.5f * scrollState.value
-                        }
-                        .background(PRIMARY_BLACK)
-                        .fillMaxWidth()
-                        .height(60.dp)
-                ) {
-                    Text(
-                        text = "CLOSED",
-                        fontWeight = FontWeight(500),
-                        fontSize = 16.sp,
-                        lineHeight = 19.5.sp,
-                        textAlign = TextAlign.Center,
-                        letterSpacing = 3.sp,
-                        color = Color.White,
-                        fontFamily = montserratFamily,
-                        modifier = Modifier.align(Alignment.Center)
-                    )
+            Surface(
+                shape = CircleShape,
+                color = Color.White,
+                modifier = Modifier
+                    .size(104.dp)
+                    .offset(y = 51.dp)
+                    .align(Alignment.BottomCenter)
+                    .graphicsLayer {
+                        translationY = -0.5f * scrollState.value.toFloat()
+                    }
+            ) {
+                Box(modifier = Modifier.fillMaxSize()) {
+                    if (gym != null && (gym!!.hours[day] == null || !isOpen(gym!!.hours[day]!!))) {
+                        Text(
+                            text = "Closed",
+                            fontWeight = FontWeight(700),
+                            fontSize = 14.sp,
+                            lineHeight = 17.07.sp,
+                            textAlign = TextAlign.Center,
+                            color = ACCENT_CLOSED,
+                            fontFamily = montserratFamily,
+                            modifier = Modifier
+                                .align(Alignment.TopCenter)
+                                .offset(y = 25.dp)
+                        )
+                    }else{
+                        Text(
+                            text = "Closed",
```

Comment:
I assume this was supposed to be `"OPEN"`? And the designs indicate that this and `"CLOSED"` should be all capitalized.

Reply:
Also, run a linter on the codebase. The spacing is off.